### PR TITLE
Default Roles and advanced policy settings UI

### PR DESCRIPTION
**Pull request by maya-barak**

### CODE REVIEW

File: docs/how-to/build-policies/policy-basics.mdx [hunk 1]
```diff
@@ -85,6 +85,19 @@ resource.
 _Above is a Policy Editor example with two roles - **Admin** & **Supervisor** - and three resources `Board`, `Task`, `Document`_.
 _As an example, the Admin cannot sign a Document._
 
+### Default Settings
+
+By default, every new resource will have default roles assigned to it: Admin, Editor, and Viewer.
+These roles come with default actions (if they exist in the resource action). You can edit the default roles and actions based on your needs.
```
Comment (GonPessach) [inline docs/how-to/build-policies/policy-basics.mdx:91]: - The name of the feature is "default permissions" 
- What do you mean by "if they exist in the resource action" ? as the default permission **create these actions** on the new resource.

Comment (GonPessach) [reply docs/how-to/build-policies/policy-basics.mdx:91]: "every new resource will have default roles assigned to it" 
the roles are not assigned to the resource, but the actions are. 
the roles have default permission (can they / can't they perform the assigned role actions) 

so the explanation maybe has two parts: 
1. the actions will appear by default on every new resource. 
2.  Roles and permissions will automatically be added to **the policy** of every new resource in the environment

File: docs/how-to/build-policies/policy-basics.mdx [hunk 2]
```diff
@@ -85,6 +85,19 @@ resource.
...
+To disable "Assign the Default Roles to every new resource," visit the **[advanced settings](https://app.permit.io/settings/advanced)** section.
+
+![Default Settings](/img/default-setting-form.png)
```
Comment (GonPessach) [inline docs/how-to/build-policies/policy-basics.mdx:94]: change screenshot after fixing UI issues

**Review by RazcoDev — APPROVED** [review]
👍

**Pull request merged**